src/components/context/DataProvider.tsx

```diff
@@ -93,8 +93,11 @@ const DataProvider = ({ children }: Props): JSX.Element => {
 
   // todo: have a dynamic value
   const requestedSampleSize = DEFAULT_REQUEST_SAMPLE_SIZE;
-  const { data: descendants = [] } = hooks.useDescendants({
+  const { data: appDescendants = [] } = hooks.useDescendants({
     id: itemId || '',
+    types: [ItemType.APP],
+    // should we show hidden elements ??
```

Comment:
Remove the comment and create an issue? But indeed I think it makes sense to not show hidden items.